### ISSUE TITLE
"Expression parsing and evaluation" typo

### ISSUE THREAD

**Issue by Bosch-Eli-Black** [josdejong/mathjs#2959]
**Describe the bug**
The documentation at https://mathjs.org/docs/expressions/parsing.html#parser has this example:

```javascript
// create a parser
const parser = math.parser()

...

// define variables and functions
parser.evaluate('x = 7 / 2')            // 3.5
parser.evaluate('x + 3')                // 6.5
parser.evaluate('f(x, y) = x^y')        // f(x, y)
parser.evaluate('f(2, 3)')              // 8

// get and set variables and functions
const x = parser.get('x')               // x = 7
```

Shouldn't the last line return `3.5` instead of `7`?

e.g.
```javascript
const x = parser.get('x')               // x = 3.5
```

**Comment by josdejong** [comment]
Thanks for reporting. Fixed via #2963 (not yet published)

**Comment by Bosch-Eli-Black** [comment]
Thanks! 🙂
